suite-native/app/index.js

```diff
@@ -1,15 +1,11 @@
 import 'react-native-gesture-handler';
 import './globalPolyfills';
-import { AppRegistry } from 'react-native';
 
-import { name as appName } from './app.json';
-import { App } from './src/App';
+import { registerRootComponent } from 'expo';
 
-if (__DEV__) {
-    // Flipper plugin for debugging websocket traffic.
-    // more: https://github.com/Matju-M/flipper-plugin-basil-ws
-    // eslint-disable-next-line chai-friendly/no-unused-expressions
-    require('basil-ws-flipper').wsDebugPlugin;
-}
```

Comment:
Will Flipper become unavailable from now on in favor of new react-native debugger?

Reply:
I tried it and it works somehow, but I did not test it extensively.